### DISCUSSION TITLE
Is anyone else inable to signup at the moment?

### DISCUSSION THREAD

**Discussion by MrChadMWood** [Budibase/budibase#12148] [Help] [answered]
On the website while trying to make a cloud account, I keep getting several errors, such as:

`Error: account --- does not exist.`

```
<div class="toolbar"><div class="devtools-separator"></div><div class="devtools-searchbox"><input class="searchBox devtools-filterinput" placeholder="Filter JSON" value=""></div></div><div class="panelContent" id="json-scrolling-panel" tabindex="0">

  |  
-- | --
message | "Name: '---'. Only lowercase characters (a-z), digits (0-9), and any of the characters _, $, (, ), +, -, and / are allowed. Must begin with a letter."
status | 400

</div>
```
`Error: Error creating tenant: {"message":"Cannot read properties of undefined (reading 'value')","status":400}`

I'm not doing anything strange that I can think of... I've tried signing up with several gmails, and a private email. Nothing seems to work at the moment.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @MrChadMWood,

It appears there was a sign-up issue over the weekend, but I'm pleased to inform you that it has been resolved. To complete the sign-up process and use your account, we may need to delete the account associated with the email you used.

Can you firstly try to login to your account. If the error still persists can you email into `support@budibase.com` with the email address used to create the account, also mention in the email that you are happy for us to delete it for you.

Once this has been deleted you should be able to go through the sign-up process again without issues.

I hope this helps.
